esmvaltool/recipes/recipe_ocean_Landschutzer2014.yml

```diff
@@ -0,0 +1,131 @@
+# ESMValTool
+# Landschutzer2014.yml
+---
+documentation:
+  description: |
+    Recipe to evaluate CO2 Fluxes of marine biogeochemistry models of CMIP5
+    using Landschutzer et al (2.014) dataset.
+    Written by Tomas Lovato, CMCC, tomas.lovato@cmcc.it
+
+  authors:
+    - lova_to
+  
+  maintainer:
+    - lova_to
+
+  references:
+    - acknow_project
+
+  projects:
+    - crescendo
+
+
+datasets:
+# working datasets
+  - {dataset: CanESM2,       project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
+#  - {dataset: HadGEM2-ES,    project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
+#  - {dataset: GISS-E2-H,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadGEM2-AO,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadCM3,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CanCM4,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CSIRO-Mk3-6-0,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+
+
+# --------------------------------------------------
+# Preprocessors
+# --------------------------------------------------
+preprocessors:
+  # --------------------------------------------------
+  # map preprocessors
+  # --------------------------------------------------
+  # For a 2D global surface map
+  prep_surface_map_2D:
+    time_average:
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+
+  # Global area-weighted Average from 2D field
+  prep_global_Surface_average_timeseries_2D:
+    custom_order: true
+    average_region:
+      coord1: longitude
+      coord2: latitude
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+# --------------------------------------------------
+# Diagnostics
+# --------------------------------------------------
+diagnostics:
+
+  # --------------------------------------------------
+  # Surface time series vs OBS
+  # --------------------------------------------------
+  diag_timeseries_surface_average_vs_OBS:
+    description: Global surface time series (Landschutzer2014 observations)
+    variables:
+      dpco2:
+        preprocessor: prep_global_Surface_average_timeseries_2D
+        field: TO2M
+        mip: Omon
+        fx_files: [areacello, ]
+      spco2:
+        preprocessor: prep_global_Surface_average_timeseries_2D
+        field: TO2M
+        mip: Omon
+        fx_files: [areacello, ]
+      fgco2:
+        preprocessor: prep_global_Surface_average_timeseries_2D
+        field: TO2M
+        mip: Omon
+        fx_files: [areacello, ]
+    additional_datasets:
```

Comment:
Why is this in `addtional_datasets`? It looks like you use it everywhere.

Reply:
@bouweandela It is in there to produce individual map plot of observations along with models.

Reply:
Yes, but why is it not in the same list as all other datasets?

Reply:
@bouweandela I followed as general example the `recipe_ocean_bgc.yml` recipe ... I think we can go along with that ...